src/Microsoft.Diagnostics.Monitoring.WebApi/Models/EgressOperationStatus.cs

```diff
@@ -20,6 +20,21 @@ public class OperationSummary
 
         [JsonPropertyName("status")]
         public OperationState Status { get; set; }
+
+        [JsonPropertyName("processInfo")]
+        public OperationProcessInfo ProcessInfo { get; set; }
+    }
+
+    public class OperationProcessInfo
```

Comment:
Need to add to documentation.

Reply:
I may have misinterpeted @wiktork 's ask here, but I was assuming that this was in reference to needing to update our docs on GitHub (so that people know the operations endpoint now includes `ProcessID`, `UID`, and `ProcessName`)

Reply:
Missed that lol. Just updated the API documentation for the list operations and get operations options.